fuzz/Cargo.toml

```diff
@@ -13,6 +13,7 @@ libfuzzer-sys = { version = "0.3" }
 fuzz_helper = { path = "fuzz_helper" }
 # This import explicitly locks the version.
 serde_json = { version = "=1.0.69" }
+bumpalo = "=3.8.0"
```

Comment:
This should ideally be done in the `Cargo.lock` file, but we probably don't have it committed. So I would at least add a comment as to why we add a transitive dependency as a direct dependency, and that we should remove this once CTAP is a library and we can finally run fuzzing with a normal compiler.

Reply:
Done, in all files that need it.